README.md

```diff
@@ -59,8 +59,78 @@ rdataretriever::download('portal', './data/')
 portal = rdataretriever::fetch('portal')
 names(portal)
 head(portal$species)
+
 ```
 
+New Spatial data Installation
+-----------------------------
+
+**Set-up and Requirements(Tools)**
+
+**Tools**
+
+-  psql, raster2pgsql, shp2pgsql, dgal,
+
+The `rdataretriever` supports installation of spatial data into `Postgres DBMS`.
+
+1. **Install Postgres**
+
+	To install `Postgres` from scratch please ref to [Postgres installation](https://trac.osgeo.org/postgis/wiki/UsersWikiPostGIS21UbuntuPGSQL93Apt).
+	For simplicity, use `.pgpass` file to avoid supplying the password every time.
+
+	After installation, Make sure you have the paths to these tools added to your system's `PATHS`.
+
+	Note: Older version of this `raster2pgsql` was a python script that you had to download and manually include in Postgres's  directory.
```

Comment:
If `raster2pgsql` is automatically installed with newer versions I might add that information here. E.g., If you are doing a new installation of PostGIS then the `raster2pgsql` requirement will be installed automatically. If you have an existing installation then in some cases you may need to ..."

Right now it's not clear what the user should do with this information. It might also be fine to remove it.